js/src/mpl.js

```diff
@@ -160,7 +157,7 @@ mpl.figure.prototype._init_canvas = function() {
 
     var pass_mouse_events = true;
 
-    canvas_div.resizable({
+    /*canvas_div.resizable({
```

Comment:
Either removed, or I should put a WIP in the PR description and iterate to make this work properly.